plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -148,6 +150,9 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
                     return Optional.of(varcharColumnMapping(createUnboundedVarcharType(), true));
                 }
                 return Optional.of(defaultVarcharColumnMapping(columnSize, true));
+            case Types.TIMESTAMP:
+                // TODO: use {@link #timestampColumnMapping} when https://issues.apache.org/jira/browse/CALCITE-1630 gets resolved
+                return Optional.of(timestampColumnMappingUsingSqlTimestampWithRoundingFullPushdown(TIMESTAMP_MILLIS));
```

Comment:
Does Druid always have milliseconds or does it support variable precision too?

Reply:
I think it always have milliseconds:

>Timestamps (including the __time column) are treated by Druid as longs, with the value being the number of milliseconds since 1970-01-01 00:00:00 UTC, not counting leap seconds.

from https://druid.apache.org/docs/latest/querying/sql.html#standard-types

>Time functions can be used with Druid's __time column, with any column storing millisecond timestamps through use of the MILLIS_TO_TIMESTAMP function, or with any column storing string timestamps through use of the TIME_PARSE function.

from https://druid.apache.org/docs/latest/querying/sql.html#time-functions